client/blocks/jetpack-review-prompt/index.tsx

```diff
@@ -74,21 +73,17 @@ const JetpackReviewPrompt: FunctionComponent< Props > = ( { align = 'center', ty
 			case 'restore':
 				return (
 					<p>
-						{ preventWidows(
-							translate(
-								'Was it easy to restore your site? Leave us a review and help spread the word.'
-							)
+						{ translate(
+							'Was it easy to restore your site? Leave us a review and help spread the word.'
 						) }
 					</p>
 				);
 			case 'scan':
 			default:
 				return (
 					<p>
-						{ preventWidows(
-							translate(
-								'Are you happy with Jetpack Scan? Leave us a review and help spread the word.'
```

Comment:
In Jetpack Cloud env, go to backups, restore a previous version and see prompt pop up after restore:

<img width="729" alt="Screenshot 2025-05-07 at 1 43 13 pm" src="https://github.com/user-attachments/assets/cb8fd490-a3e2-4978-9f4a-82996412b828" />

Reply:
Not someting for this PR, and not urgent, but it'd be nice to give this modal a DS pass. CC: @keoshi in case you would know where to file this.